modules/userId/index.js

```diff
@@ -909,6 +919,16 @@ function updateInitializedSubmodules(dest, submodule) {
   }
 }
 
+function getConfiguredStorageTypes(config) {
+  return config?.storage?.type?.split('&') || [];
```

Comment:
Since we're parsing config.storage.type as a boolean logic statement instead of an enum, I think we should provide 
 allowance for optional whitespace -- both leading and trailing whitespace and whitespace around the ampersand. So we should `trim()` the type and split on `/\s*&\s*/`.